### ISSUE TITLE
Security vulnerability

### ISSUE THREAD

**Issue by sarthakpati** [CBICA/BrainMaGe#18]
According to dependabot:

![image](https://user-images.githubusercontent.com/11719673/164896000-aef14961-f95a-44cf-8b53-8ddc217f92de.png)

Link to patch: https://github.com/pytorchlightning/pytorch-lightning/commit/8b7a12c52e52a06408e9231647839ddb4665e8ae

PyTorch lightning 1.6.1 is already out [[ref](https://github.com/PyTorchLightning/pytorch-lightning/tree/1.6.1)] which should contain this fix, but making any change in dependencies would require testing.

**Comment by Geeks-Sid** [comment]
yeah, and i have seen 1.6.1 break my code, so i might need to rewrite multiple things. Keeping it open for now, will come back later.

**Comment by sarthakpati** [comment]
Getting addressed in #19.
